contrib/google.golang.org/grpc/option.go

```diff
@@ -68,6 +78,21 @@ func NoDebugStack() Option {
 	}
 }
 
+// NonErrorCodes marks the given codes to be treated as non-errors.
+// codes.OK will always be treated as a success.
```

Comment:
This line is not needed. I think it is already expected that `codes.OK` should not be treated as an error.